python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2391,6 +2391,7 @@ def _impl(inputs, attr, params, mod):
     "ReverseV2": _reverse_v2(),
     "RightShift": AttrCvt("right_shift"),
     "Round": AttrCvt("round"),
+    "Rint": AttrCvt("round"),
```

Comment:
preserve the alphabetical order. keep Rint above Round.

Reply:
🆗